modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v10_4_1/ObjectDefinitionUpgradeProcess.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.internal.upgrade.v10_4_1;
+
+import com.liferay.petra.string.StringBundler;
+import com.liferay.portal.dao.orm.common.SQLTransformer;
+import com.liferay.portal.kernel.dao.jdbc.AutoBatchPreparedStatementUtil;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+/**
+ * @author Alberto Sousa
+ */
+public class ObjectDefinitionUpgradeProcess extends UpgradeProcess {
```

Comment:
Please rename to `ObjectDefinitionStaleClassNameUpgradeProcess`.
Update the test class.